test/core/api/service/author/author_api_test.cpp

```diff
@@ -513,3 +513,31 @@ TEST_F(AuthorApiTest, SubmitAndWatchExtrinsicSubmitsAndWatches) {
                          author_api->submitAndWatchExtrinsic(*extrinsic));
   ASSERT_EQ(sub_id, ret_sub_id);
 }
+
+/**
+ * @when requesting list of extrinsics
+ * @then extrinsics are fetched from transaction pool and returned as a vector
+ */
+TEST_F(AuthorApiTest, PendingExtrinsics) {
+  std::unordered_map<Transaction::Hash, std::shared_ptr<Transaction>> trxs;
+  std::vector<Extrinsic> expected_result;
+
+  EXPECT_CALL(*transaction_pool, getPendingTransactions())
+      .WillOnce(ReturnRef(trxs));
+
+  ASSERT_EQ(expected_result, author_api->pendingExtrinsics().value());
```

Comment:
Please, extract statement to ASSERT_OUTCOME_SUCCESS
```suggestion
  ASSERT_OUTCOME_SUCCESS(actual_result, author_api->pendingExtrinsics());
  ASSERT_EQ(expected_result, actual_result);
```

And look at other the same cases.